============================= test session starts ==============================
platform linux -- Python 3.10.12, pytest-8.3.2, pluggy-1.5.0
rootdir: /home/bridge/code/incubyte
plugins: Faker-24.4.0, anyio-4.4.0
collected 9 items

test.py ...s....F                                                        [100%]

=================================== FAILURES ===================================
_________________ TestAddFunction.test_prevent_4_digit_numbers _________________

self = <test.TestAddFunction object at 0x7019a3215bd0>

    def test_prevent_4_digit_numbers(self):
        response = add('//;\n1;2;1000;3')
>       assert response == 6
E       assert 1006 == 6

test.py:48: AssertionError
=========================== short test summary info ============================
FAILED test.py::TestAddFunction::test_prevent_4_digit_numbers - assert 1006 == 6
==================== 1 failed, 7 passed, 1 skipped in 0.04s ====================
